docs/web-servers/nginx/enable-tls-on-nginx-for-https-connections.md

```diff
@@ -0,0 +1,186 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'Install a TLS certificate into NGINX for HTTPS access.'
+keywords: ["ssl", "tls", "nginx", "https", "certificate"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+aliases: ['web-servers/nginx/configuration/ssl/','security/ssl/ssl-certificates-with-nginx/','security/ssl/how-to-provide-encrypted-access-to-resources-using-ssl-certificated-on-nginx/','security/ssl/provide-encrypted-resource-access-using-ssl-certificates-on-nginx/']
+modified: 2018-01-09
+modified_by:
+  name: Linode
+published: 2018-01-09
+title: 'Getting Started with NGINX - Part 3: Enable TLS for HTTPS Connections'
+---
+
+![HTTPS Configuration on NGINX](/docs/assets/nginx-ssl/Enable_SSL_nginx.jpg)
```

Comment:
@cwlinode should this graphic be updated to match the title? currently reads "Enable SSL for HTTPS  Configuration on NGiNX"

Reply:
Actually, if you could please remove that line so the graphic doesn't show, I'll update that image and add add'l ones after publish.

TIA.